### ISSUE TITLE
Incorrectly remove image when path has accent

### ISSUE THREAD

**Issue by Ryuno-Ki** [mdn/yari#9445]
### Summary

This report was originally filled against content: https://github.com/mdn/content/issues/28341

https://github.com/mdn/content/pull/28233 removed images that were allegedly unused but now yield a broken reference on an SVG page containing Bézier curves.

While triaging I confirmed that the references exist, are relative to the directory (so structurally equal to other image references on the page) but show up encoded with b%C3%A9zier_curves in the URL.

### URL

https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Paths

### Reproduction steps

1. Visit https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Paths#b%C3%A9zier_curves
2. Scroll down to the images showing the impact on changes
3. Notice a broken image

### Expected behavior

The images should be displayed.

### Actual behavior

Broken image with alt-text

### Device

Desktop

### Browser

Chrome

### Browser version

Stable

### Operating system

Linux

### Anything else?

I'm not sure which device the original reporter used. I can confirm with Chromium on Linux Desktop (since Firefox crashes too often for me at the moment).

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions/categories/platform).

**Comment by bsmth** [comment]
Good catch, so `yarn filecheck` doesn't yet check if `bézier_curves` == `b%C3%A9zier_curves`?

**Comment by caugner** [comment]
It looks like this is no longer an issue.

The French [Bézier curve](https://developer.mozilla.org/en-US/docs/Glossary/Bezier_curve) glossary page includes an image named `bézier_2_big.gif`. I checked [the last PR](https://github.com/mdn/translated-content/pull/19645) that touched that translated page, and the Review companion comment did not include a flaw related to the accent.
